testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/resource/param/MultiValuedParamCdiTest.java

```diff
@@ -0,0 +1,121 @@
+package org.jboss.resteasy.test.resource.param;
+
+
+import org.jboss.arquillian.container.test.api.Deployment;
+import org.jboss.arquillian.container.test.api.RunAsClient;
+import org.jboss.arquillian.junit.Arquillian;
+import org.jboss.resteasy.client.jaxrs.ResteasyClient;
+import org.jboss.resteasy.client.jaxrs.ResteasyClientBuilder;
+import org.jboss.resteasy.test.resource.param.resource.MultiValuedParamCdiResource;
+import org.jboss.resteasy.test.resource.param.resource.MultiValuedParamPersonConverterProvider;
+import org.jboss.resteasy.test.resource.param.resource.MultiValuedParamPersonArrayConverter;
+import org.jboss.resteasy.test.resource.param.resource.MultiValuedParamPersonListConverter;
+import org.jboss.resteasy.test.resource.param.resource.MultiValuedParamPersonSetConverter;
+import org.jboss.resteasy.test.resource.param.resource.MultiValuedParamPersonSortedSetConverter;
+import org.jboss.resteasy.test.resource.param.resource.MultiValuedParamPersonWithConverter;
+import org.jboss.resteasy.utils.PortProviderUtil;
+import org.jboss.resteasy.utils.TestUtil;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.ws.rs.core.Response;
+
+/**
+ * @tpSubChapter Parameters
+ * @tpChapter Integration tests
+ * @tpTestCaseDetails Test of CDI integration for RESTEasy extended support for multivalue @*Param (RESTEASY-1566 + RESTEASY-1746)
+ * @tpSince RESTEasy 3.6.1.Final
```

Comment:
@asoldano, will the next tag on this branch be "3.6.1.SP0"?

Reply:
Looking to [3.0.19.SPx branch](https://github.com/resteasy/Resteasy/commits/3.0.19.SPx), the first SP release was [3.0.19.SP1](https://github.com/resteasy/Resteasy/commit/ac346d5eae9fc9e7cddede1c6987504cd809dbea), so the correct version here should probably be 3.6.1.SP1 ...

Reply:
https://github.com/resteasy/Resteasy/pull/1660#issuecomment-413489886